tools/sigclient/datafiles/SPL_Playground.csv

```diff
@@ -0,0 +1,119 @@
+* | search hobby=Walking
+search app_name=Codhave
+search http_status>400
+http_status=20*
+* | hobby=Act*
+"""* | hobby=""""Slot car*"""""""
+"""search app_name=Martencan address=""""157 Overpass berg, Buffalo, Alaska 51094"""""""

```

Comment:
Why do you need so many quotes here? I thought `""` would escape the `"` so you could just do:
```
"* | hobby=""Slot car*"""
```
which—after escaping—would effectively be:
```
* | hobby="Slot car*"
```